src/content/testing/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Okteto Test
+description: In this section, we'll see how you can use Test Containers on Okteto
+---
+
+# Welcome to Okteto Test
+
+Okteto Test is a new feature designed to seamlessly integrate testing into your development workflow on the Okteto platform. Whether you're running unit tests or end-to-end (E2E) tests, Okteto Test ensures an efficient testing experience before even making a pull request.
+
+## Benefits of Okteto Test
+
+### 1. Seamless Integration
+Developers can execute tests in the same production-like Kubernetes environment where their code runs.
+This ensures consistency and reduces configuration overhead.
+
+### 2. Simplified Configuration
+The new `test` section in the [Okteto manifest](reference/okteto-manifest.mdx#test-object-optional) supports multiple Test Containers, allowing you to define your test environment with minimal YAML configuration.
+This eliminates the need for complex setups and advanced Kubernetes knowledge.
+
+### 3. Faster Feedback Loop
```

Comment:
What about adding a forth point to talk about remote execution? We can use it to say that developers don't need to install anything locally (test tools are packaged in the test container image). And running the tests in the cluster network simplify access to private services and reduce latency